client/src/api/pitch/index.ts

```diff
@@ -59,7 +60,7 @@ const declinePitch = async (
 
 // Updates the information on a pitch
 const updatePitch = async (
-  pitchData: { [key: string]: string[] | string | number | IPitch['teams'] },
+  pitchData: { [K in keyof IPitch]?: IPitch[K] },
```

Comment:
I believe this type could be replaced with `Partial<IPitch>`. I might be wrong though, you could give it a try.

Reply:
Ur a genius Andrew